packages/sommelier.rb

```diff
@@ -28,7 +28,7 @@ class Sommelier < Package
   depends_on 'libxcvt'
   depends_on 'libxfixes' => :build
   depends_on 'libxkbcommon' # R
-  depends_on 'llvm' # R
+  depends_on 'llvm' # R Note that this may need rebuilds for newer llvm versions.
```

Comment:
It would be nice if sommelier didn't depend on llvm.  We need to figure out a way to have a minimal install without all the compiler tools since that package is so huge.  Also, it doesn't make sense to require it as a runtime dependency.  It should be a build only tool.  Maybe create a `libllvm` package to be used as a runtime dependency?